.github/actions/setup-nvidia/action.yml

```diff
@@ -140,11 +148,107 @@ runs:
               )
           }
 
+          install_nvidia_driver_ubuntu20() {
```

Comment:
Looks like it shares a lot of code with `install_nvidia_driver_amzn2`, can parts of it can be refactored into a smaller routines and reused?

Reply:
Yes, will work on refactoring and then test both cases (install_nvidia_driver_amzn2) and (install_nvidia_driver_ubuntu20)